### PR TITLE
Add 69shu and 69xinshu

### PR DESCRIPTION
**Pull request by he0119**
Also use NovelStatus and translate filter labels in linovelib.

### CODE REVIEW

File: icons/src/cn/69shu/icon.png
Comment (nyagami) [inline icons/src/cn/69shu/icon.png:1]: lul, isnt it too small

File: plugins/chinese/69shu.ts [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+import { Plugin } from "@typings/plugin";
+import { NovelStatus } from "@libs/novelStatus";
+
+class Shu69 implements Plugin.PluginBase {
+    id = "69shu";
+    name = "69书吧";
+    icon = "src/cn/69shu/icon.png";
+    site = "https://www.69shu.xyz";
+    version = "0.1.0";
+    userAgent = "";
```
Comment (nyagami) [inline plugins/chinese/69shu.ts:13]:
```suggestion
```

File: plugins/chinese/69xinshu.ts [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+import { encode } from "urlencode";
+import { NovelStatus } from "@libs/novelStatus";
+
+class XinShu69 implements Plugin.PluginBase {
+    id = "69xinshu";
+    name = "69书吧";
+    icon = "src/cn/69xinshu/icon.png";
+    site = "https://www.69xinshu.com";
+    version = "0.1.0";
+    userAgent = "";
```
Comment (nyagami) [inline plugins/chinese/69xinshu.ts:14]:
```suggestion
```

**Comment by nyagami** [comment]
Also I see that 69xinshu has release date in chapter list. If will be better if you can parse them as ISO strings for each chapter (chapter.releaseTime)

File: icons/src/cn/69shu/icon.png
Comment (he0119) [reply icons/src/cn/69shu/icon.png:1]: couldn't find larger one😂

**Comment by he0119** [comment]
> Also I see that 69xinshu has release date in chapter list. If will be better if you can parse them as ISO strings for each chapter (chapter.releaseTime)

Only latest chapters have release date.

**Review by nyagami — APPROVED** [review]

**Pull request merged**